website/source/docs/job-specification/resources.html.md

```diff
@@ -49,7 +49,7 @@ job "docs" {
 - `iops` `(int: 0)` - Specifies the number of IOPS required given as a weight
   between 0-1000.
 
-- `memory` `(int: 300)` - Specifies the memory required in MB
+- `memory` `(int: 10)` - Specifies the memory required in MB
```

Comment:
Can we actually change this to be 300 instead for changing the docs: https://github.com/hashicorp/nomad/blob/master/api/resources.go#L20